<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QMainWindow" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome New User!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>USERNAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>170</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PASSWORD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>207</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CONFIRM PASSWORD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_reg">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>371</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_reg">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>128</y>
      <width>371</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_reg">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>164</y>
      <width>371</width>
      <height>27</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="confpass_reg">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>202</y>
      <width>371</width>
      <height>27</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="regButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>270</y>
      <width>251</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Back To Login Window</string>
    </property>
   </widget>
   <widget class="QLabel" name="reg_error_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
